packages/design-system/src/atoms/popover/index.web.tsx

```diff
@@ -69,7 +69,6 @@ export const Popover = ({
       {renderReference({
         reference: refs.setReference,
         getReferenceProps,
-        selectedValue,
```

Comment:
Popover is not something that is selectable. This prop is only used in Select, so I moved it there.